source/__tests__/integration/create-resource.test.js

```diff
@@ -52,9 +52,9 @@ describe('Create profile resource template requirements', () => {
         resourceURI: "htt",
       }).catch(error => console.log(`promise error for filling profile form: ${error}`));
       await page.waitFor(1000)
-      page
-        .waitForSelector('input#resourceURI.ng-invalid-url')
-        .catch(error => console.log(`promise error checkURL: ${error}`))
+      const invalid_url_class = page.$('input#resourceURI', e => e.getAttribute('ng-invalid-url'))
+      expect(invalid_url_class).toBeDefined()
```

Comment:
this is fine this way.  It's okay also to wait for selector, with the catch message, AND have the expect afterwards.  I'm not sure if `page.blah` waits the way `waitForSelector` does.